co2eq/purchase/footprints.yml

```diff
@@ -28,6 +28,134 @@ _children:
           - PURCHASE_CATEGORY_TRANSPORTATION_AUTOMOTIVE_PARKING
         source: 'http://www.balticproject.org/en/calculator-page'
         unit: EUR
+  Appliances:
+    _children:
+      'Electric Oven':
```

Comment:
sort alphabetically?

Reply:
I don't think that's a good idea - now they're grouped by subcategories which I think makes more sense.